src/llmcompressor/transformers/compression/helpers.py

```diff
@@ -30,6 +35,9 @@ def tensor_follows_mask_structure(tensor, mask: str = "2:4") -> bool:
         atleast n zeros in each chunk of size m
     """
 
+    if mask.lower().strip() == "unstructured":
```

Comment:
nit: may want to make a note about this acceptable mask format in the docstring

Reply:
nit: missing type hint for `tensor`

Reply:
Updated!